templates/etc/icinga2/constants.conf.j2

```diff
@@ -0,0 +1,32 @@
+/**
+ *  {{ ansible_managed }}
```

Comment:
Should we add a comment from which icinga2 package version the initial config file was taken? Thus it's easier to compare them later on in operations.

Reply:
Is probably a good idea